### ISSUE TITLE
include releases on github

### ISSUE THREAD

**Issue by lastlink** [unified-doc/unified-doc#1]
when you publish new npm release could you also add them to github? I found out about this as I was reviewing https://www.npmjs.com/package/@funktechno/texthighlighter.

**Comment by chrisrzhou** [comment]
Will do @lastlink! The project is in beta, so still formalizing and locking in the core APIs before they are formally maintained.  I expect to get the core formalized in the next two weeks.

**Comment by chrisrzhou** [comment]
@lastlink, official `v1.0.0` release is out! In the upcoming days, I'll be updating other repos and docs.  The APIs for `v1.0.0` should be fairly stable in the near future.

This is my first time publishing release on Github and working with a monorepo.  I noticed that the compressed source code includes all packages, so I'm not sure how to only point that to the specific package in the monorepo.

I took a quick reference to the https://github.com/rehypejs/rehype/releases project and notice this is the same behavior.  In the future, I'll figure out how to attach build files to the Github releases, but until then I think this is the right way of organizing this for a monorepo.
